maps-app/src/main/res/values/app_settings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- TODO: add your OAuth Client ID here-->
+    <string name="client_id"></string>
+
```

Comment:
@zinfin This will resolve the missing app_settings.xml file and subsequent resolution of `client_id`.